proto/sge/strategicreserve/proposal.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+package sgenetwork.sge.strategicreserve;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/sge-network/sge/x/strategicreserve/types";
+
+// DataFeeCollectorFeedProposal details a proposal for use bet and house fee,
+// together with how many tokens are proposed to be transfered.
+message DataFeeCollectorFeedProposal {
```

Comment:
We may decide to do this in end blocker based on some business logic or threshold.

Reply:
Module account removed #153